### PR TITLE
Create Sys_7045_High_LateralMovement-SMBEXEC.yml

### PR DESCRIPTION
**Pull request by longmdx**
Lateral Movement with SMBEXEC.

### CODE REVIEW

**Comment by YamatoSecurity** [comment]
@longmdx Thank you very much for the PR. We will test it and get back to you as soon as possible.

**Review by fukusuket — COMMENTED** [review]
@longmdx 
Thank you so much for the PR :)
The following detection rules for SMBExec already exist, are these rules insufficient?
- https://github.com/Yamato-Security/hayabusa-rules/blob/main/sigma/builtin/system/service_control_manager/win_system_hack_smbexec.yml
- https://github.com/SigmaHQ/sigma/blob/95793d73bd34b27653bc2f55cb80db402655ea01/rules/windows/builtin/system/service_control_manager/win_system_hack_smbexec.yml

**Comment by longmdx** [comment]
@fukusuket 
Sorry, I didn't carefully check that there is a rule for SMBEXEC, but the source code of smbexec.py has changed. Now, **batchFile** is generated with the string `%SYSTEMROOT%\' + ''.join([random.choice(string.ascii_letters) for _ in range(8)]) + '.bat'` . So, using the condition `ImagePath|endswith: \execute.bat` will miss the alert for this behavior.

File: hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 author: longmd - n0n$la$
 date: 2023/10/16
-modified: 2022/10/16
+modified: 2023/10/19
 
 title: SMBExec Lateral Movement
-details: 'Svc: %ServiceName% ¦ Path: %ImagePath% ¦ Acct: %AccountName% ¦ StartType: %StartType%'
+details: 'Svc %ServiceName% was installed (%StartType%) by %AccountName%'
```
Comment (hitenkoku) [inline hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml:6]:
```suggestion
details: 'Svc: %ServiceName% ¦ Path: %ImagePath% ¦ Acct: %AccountName% ¦ StartType: %StartType%'
```

When `json-timeline` option was used  in Hayabusa, details data split by `¦` to check easily, So I think this format suggest.

File: hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 author: longmd - n0n$la$
 date: 2023/10/16
-modified: 2022/10/16
+modified: 2023/10/19
 
 title: SMBExec Lateral Movement
```
Comment (hitenkoku) [inline hayabusa/builtin/System/Sys_7045_Critical_LateralMovement-SMBEXEC.yml:5]:
```suggestion
title: SMBExec Lateral Movement  (service installation)
```
With just this title, it's hard to know what exactly you are doing, so how about this way of writing?

**Review by hitenkoku — COMMENTED** [review]
Thanks for your pull request.

I have few comment. Could you check it?

@YamatoSecurity  @fukusuket I have commented on what I think, if I am wrong please feel free to point out my comment.

**Comment by fukusuket** [comment]
@longmdx 
I see!  As you said, it has been changed in this commit https://github.com/fortra/impacket/commit/35cbac53c7cf78563c3e7269bcafbc6706083f01 

If possible, how about submitting a PR directly to the [Sigma repository's smbexec rule files](https://github.com/SigmaHQ/sigma/blob/95793d73bd34b27653bc2f55cb80db402655ea01/rules/windows/builtin/system/service_control_manager/win_system_hack_smbexec.yml)? I think it will be a good PR for all Sigma users!(not only Hayabusa user) All changes made to the [Sigma repository](https://github.com/SigmaHQ/sigma) are automatically applied to Hayabusa on a daily basis.

@YamatoSecurity @hitenkoku  
I think it's better to create a PR in the Sigma repository. What do you think?

**Comment by hitenkoku** [comment]
@fukusuket I agree your comment!

**Comment by longmdx** [comment]
@fukusuket 
I appreciate your edits and feedback. Thank you for that.

**Pull request closed without merging**